### ISSUE TITLE
Alarm tone too low

### ISSUE THREAD

**Issue by maniattico** [googlesamples/assistant-sdk-python#297]
Hi.

I've Google Assistant for Device SDK in my RaspberryPI. 

My issue is with countdown and alarm sound: it is so low (all other sounds and phrases works fine).

I found that in Google Home app I can change that, however my device doen't apperars.

Are there any way to increase that volume or change the ringtone?

Thanks.

**Comment by fsnsantana** [comment]
I have the same problem, timer and alarm volume is too low and all other sounds is working fine.

**Comment by hasmar04** [comment]
@fsnantana what is your code and where did you put it? I have been trying this and I can't get it to work. It gives me a segmentation fault.

**Comment by fsnsantana** [comment]
@hasmar04 here have some instructions to handle commands on the device [https://developers.google.com/assistant/sdk/guides/library/python/extend/handle-device-commands](Google Assistant SDK)

First get the source code:
`git clone https://github.com/googlesamples/assistant-sdk-python`

Then insert your code in this file:
`cd assistant-sdk-python/google-assistant-sdk/googlesamples/assistant/library`
`nano hotword.py`

In this case, you need to create an `if event.type == EventType.ON_ALERT_STARTED:` inside `def process_event(event):` and put your code there.

Finally, start assistant from this file.

**Comment by hasmar04** [comment]
@fsnsantana do you have any working code to insert here? I have been trying my own here and the sounds always clash with the alarm sound and give segmentation error.

**Comment by fsnsantana** [comment]
@hasmar04 No, I don't. I used `assistant.stop_conversation()` to suspend assistant and initiate some custom commands at the beginning of my project, now I'm using custom device actions [Register Custom Device Actions](https://developers.google.com/assistant/sdk/guides/library/python/extend/custom-actions). In that way, it's not necessary to suppress the assistant.

But this alarm issue seems to be inside the assistant code.
